checker/check_result.go

```diff
@@ -16,30 +16,199 @@ package checker
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged